<ui version="4.0" >
 <class>JobForm</class>
 <widget class="QWidget" name="JobForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>855</width>
    <height>757</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>40</y>
     <width>231</width>
     <height>181</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title" >
    <string>Basic Information</string>
   </property>
   <widget class="QWidget" name="layoutWidget" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>211</width>
      <height>152</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout" >
     <property name="labelAlignment" >
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="formAlignment" >
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <item row="0" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>JobId:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLabel" name="label_JobId" >
       <property name="text" >
        <string>2</string>
       </property>
       <property name="textInteractionFlags" >
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Job Name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLabel" name="label_Name" >
       <property name="text" >
        <string>Test</string>
       </property>
       <property name="textInteractionFlags" >
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Level:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QLabel" name="label_Level" >
       <property name="text" >
        <string>VirtualFull</string>
       </property>
       <property name="textInteractionFlags" >
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0" >
      <widget class="QLabel" name="label_28" >
       <property name="text" >
        <string>Client:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" >
      <widget class="QLabel" name="label_Client" >
       <property name="text" >
        <string>client-fd</string>
       </property>
       <property name="textInteractionFlags" >
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0" >
      <widget class="QLabel" name="label_7" >
       <property name="text" >
        <string>FileSet:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" >
      <widget class="QLabel" name="label_FileSet" >
       <property name="text" >
        <string>TheFileSet</string>
       </property>
       <property name="textInteractionFlags" >
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item row="5" column="0" >
      <widget class="QLabel" name="label_11" >
       <property name="text" >
        <string>Pool:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" >
      <widget class="QLabel" name="label_Pool" >
       <property name="text" >
        <string>ThePool</string>
       </property>
       <property name="textInteractionFlags" >
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2" >
   <property name="geometry" >
    <rect>
     <x>680</x>
     <y>40</y>
     <width>171</width>
     <height>181</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title" >
    <string>Volume Used</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3" >
    <item>
     <widget class="QListWidget" name="list_Volume" >
      <item>
       <property name="text" >
        <string>Vol0001</string>
       </property>
       <property name="icon" >
        <iconset resource="../main.qrc" >
         <normaloff>:/images/inflag1.png</normaloff>:/images/inflag1.png</iconset>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_3" >
   <property name="geometry" >
    <rect>
     <x>420</x>
     <y>40</y>
     <width>251</width>
     <height>181</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title" >
    <string>Times</string>
   </property>
   <widget class="QWidget" name="layoutWidget" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>231</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_3" >
     <property name="fieldGrowthPolicy" >
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="labelAlignment" >
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="formAlignment" >
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <item row="0" column="0" >
      <widget class="QLabel" name="label_26" >
       <property name="text" >
        <string>Sched Time:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLabel" name="label_SchedTime" >
       <property name="text" >
        <string>2009-07-31 00:10:00</string>
       </property>
       <property name="textInteractionFlags" >
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_5" >
       <property name="text" >
        <string>Start Time:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLabel" name="label_StartTime" >
       <property name="text" >
        <string>2009-07-31 00:10:00</string>
       </property>
       <property name="textInteractionFlags" >
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="label_19" >
       <property name="text" >
        <string>End Time:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QLabel" name="label_EndTime" >
       <property name="text" >
        <string>2009-07-31 00:20:00</string>
       </property>
       <property name="textInteractionFlags" >
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0" >
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string>Duration:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" >
      <widget class="QLabel" name="label_Duration" >
       <property name="text" >
        <string>00:10:00</string>
       </property>
       <property name="textInteractionFlags" >
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4" >
   <property name="geometry" >
    <rect>
     <x>250</x>
     <y>40</y>
     <width>161</width>
     <height>181</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize" >
    <size>
     <width>160</width>
     <height>0</height>
    </size>
   </property>
   <property name="title" >
    <string>Status</string>
   </property>
   <widget class="QWidget" name="layoutWidget" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>141</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2" >
     <property name="fieldGrowthPolicy" >
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="labelAlignment" >
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="formAlignment" >
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <item row="0" column="0" >
      <widget class="QLabel" name="label_8" >
       <property name="text" >
        <string>Files:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLabel" name="label_JobFiles" >
       <property name="text" >
        <string>1,924</string>
       </property>
       <property name="textInteractionFlags" >
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_9" >
       <property name="text" >
        <string>Bytes:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="label_10" >
       <property name="text" >
        <string>Errors:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QLabel" name="label_JobErrors" >
       <property name="text" >
        <string>0</string>
       </property>
       <property name="textInteractionFlags" >
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0" >
      <widget class="QLabel" name="label_6" >
       <property name="text" >
        <string>Status:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" >
      <widget class="QLabel" name="label_JobStatus" >
       <property name="text" >
        <string/>
       </property>
       <property name="pixmap" >
        <pixmap resource="../main.qrc" >:/images/T.png</pixmap>
       </property>
       <property name="scaledContents" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0" >
      <widget class="QLabel" name="label_25" >
       <property name="text" >
        <string>Purged:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" >
      <widget class="QCheckBox" name="chkbox_PurgedFiles" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLabel" name="label_JobBytes" >
       <property name="text" >
        <string>109 MB</string>
       </property>
       <property name="textInteractionFlags" >
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>220</y>
     <width>841</width>
     <height>531</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title" >
    <string>Logs</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" >
    <item>
     <widget class="QTextEdit" name="textJobLog" >
      <property name="html" >
       <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'DejaVu Sans'; font-size:10pt; font-weight:400; font-style:normal;">
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;/p>&lt;/body>&lt;/html></string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>textJobLog</zorder>
  </widget>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>661</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" >
    <item>
     <widget class="QPushButton" name="pushButton" >
      <property name="text" >
       <string>Delete</string>
      </property>
      <property name="icon" >
       <iconset resource="../main.qrc" >
        <normaloff>:/images/purge.png</normaloff>:/images/purge.png</iconset>
      </property>
      <property name="flat" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_8" >
      <property name="toolTip" >
       <string>View errors for this Job</string>
      </property>
      <property name="text" >
       <string>Errors</string>
      </property>
      <property name="icon" >
       <iconset resource="../main.qrc" >
        <normaloff>:/images/zoom.png</normaloff>:/images/zoom.png</iconset>
      </property>
      <property name="flat" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3" >
      <property name="text" >
       <string>Media</string>
      </property>
      <property name="icon" >
       <iconset resource="../main.qrc" >
        <normaloff>:/images/zoom.png</normaloff>:/images/zoom.png</iconset>
      </property>
      <property name="flat" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_5" >
      <property name="text" >
       <string>History</string>
      </property>
      <property name="icon" >
       <iconset resource="../main.qrc" >
        <normaloff>:/images/zoom.png</normaloff>:/images/zoom.png</iconset>
      </property>
      <property name="flat" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text" >
       <string>Run again</string>
      </property>
      <property name="icon" >
       <iconset resource="../main.qrc" >
        <normaloff>:/images/R.png</normaloff>:/images/R.png</iconset>
      </property>
      <property name="flat" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_6" >
      <property name="text" >
       <string>Read doc</string>
      </property>
      <property name="flat" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_7" >
      <property name="text" >
       <string>FileSet</string>
      </property>
      <property name="icon" >
       <iconset resource="../main.qrc" >
        <normaloff>:/images/zoom.png</normaloff>:/images/zoom.png</iconset>
      </property>
      <property name="flat" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_4" >
      <property name="text" >
       <string>Stats</string>
      </property>
      <property name="icon" >
       <iconset resource="../main.qrc" >
        <normaloff>:/images/zoom.png</normaloff>:/images/zoom.png</iconset>
      </property>
      <property name="flat" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../main.qrc" />
 </resources>
 <connections/>
</ui>
